### ISSUE TITLE
Problem with BAB when importing statblocks

### ISSUE THREAD

**Issue by saturnin55** [Lavaeolous/PF1-StatBlock-Converter-Module#503]
I often have this problem when important statblocks, where the BAB is flat out wrong. In this example, it's an Ash Giant Thug from Wrath of the Righteous. I often get this bug, not sure why the parser is doing that. Thanks!

Here's the original statblock : `Ash Giant Thug

CR 12

XP 19,200

Ash giant barbarian 1 (Pathfinder RPG Bestiary 3 126)

CE Large humanoid (giant)

Init +4; Senses low-light vision; Perception +21

DEFENSE

AC 27, touch 10, flat-footed 24 (+7 armor, +3 Dex, +10 natural, -2 rage, -1 size)

hp 196 (15 HD; 14d8+1d12+121)

Fort +19, Ref +8, Will +11; +1 vs. traps

Defensive Abilities rock catching; Immune disease, poison; Resist electricity 10

OFFENSE

Spd 35 ft.

Melee +1 heavy pick +25/+20/+15 (2d6+15/19-20/x4 plus disease)

Ranged rock +16 (1d8+14 plus disease)

Space 10 ft.; Reach 10 ft.

Special Attacks disease, oversized weapon, rage (10 rounds/day), rock throwing (120 ft.)

TACTICS

During Combat The ash giant rages at the start of combat, then moves to engage in melee. He has no rocks available in this room to throw at foes.

Morale The ash giant fights to the death unless the charm monster effect placed on him by the lamia matriarch is dispelled, in which case he swiftly volunteers to aid the PCs in murdering his ex-mistress. The ash giant is good for his word as an ally until Ursathella is slain, at which point he turns on the PCs in an attempt to kill them as well. Without the false courage granted by charm monster, the ash giant flees combat if reduced to fewer than 40 hit points.

STATISTICS

Str 39, Dex 18, Con 27, Int 11, Wis 16, Cha 8

Base Atk +11; CMB +26 (+28 bull rush); CMD 38 (40 vs. bull rush)

Feats Awesome Blow, Combat Reflexes, Improved Bull Rush, Improved Critical (heavy pick), Improved Iron Will, Iron Will, Power Attack, Vital Strike

Skills Perception +21, Profession (miner) +13, Survival +16

Languages Common, Giant

SQ fast movement

Gear +1 black dragonhide electricity resistance breastplate, Huge +1 heavy pick`

![image](https://user-images.githubusercontent.com/6554852/151835926-227b3af0-ee21-4edf-bb76-8de2d9d02f69.png)

![image](https://user-images.githubusercontent.com/6554852/151836431-7e449d70-3413-432b-912e-914ea591e768.png)

**Comment by Lavaeolous** [comment]
Thanks for letting me know! I'll take a look.

**Comment by Lavaeolous** [comment]
```
Melee +1 heavy pick +25/+20/+15 (2d6+15/19-20/x4 plus disease)
```
sbc falsely thinks the +1 enhancement is the attack bonus

**Comment by Lavaeolous** [comment]
Should be fixed in 3.3.4
